go/libkb/chain_link.go

```diff
@@ -1141,3 +1167,18 @@ func (c ChainLink) NeedsSignature() bool {
 	}
 	return c.unpacked.outerLinkV2.LinkType.NeedsSignature()
 }
+
+// MaybeDropSig will erase c.unpacked.sig if it is a track link
```

Comment:
This isn't right: you can't drop the last link of a **sub**chain.  Meaning, if link 10 is a new eldest, you can't drop link 9. This is pretty hard to get right, we have a lot of machinery to check whether it's a new subchain or not.